packages/react-sdk/src/components/DeveloperTools/WhiteboardSessions.tsx

```diff
@@ -26,9 +27,18 @@ import {
 export function WhiteboardSessionsTable({
   sessions,
 }: {
-  sessions: SessionState[] | undefined;
+  sessions: (SessionState & { status?: string })[] | undefined;
 }) {
   const { t } = useTranslation('neoboard');
+  const [now, setNow] = useState(Date.now());
+  useEffect(() => {
```

Comment:
Can be replaced by `useInterval` from [react-use](https://github.com/streamich/react-use/blob/master/docs/useInterval.md)

Reply:
done